client/apps/edit/components/content/sections/text/index.coffee

```diff
@@ -319,7 +289,6 @@ module.exports = React.createClass
   getPlugins: ->
     plugins = []
     plugins.push({label: 'artist', style: 'artist'}) if @hasFollow
-    plugins.push({label: 'toc', style: 'toc'}) if @hasToc
     return plugins
```

Comment:
Seems like a good swap! I'll add that in. Are there any other text features like `ol` that are hidden atm?